signature/cose/envelope.go

```diff
@@ -0,0 +1,587 @@
+package cose
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/x509"
+	"fmt"
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-core-go/signature/internal/base"
+	"github.com/veraison/go-cose"
+)
+
+// COSE signature envelope blob mediaType
```

Comment:
The comment block should begin with `MediaTypeEnvelope ...` as per golang convension.